src/drivers/uavcan/CMakeLists.txt

```diff
@@ -130,6 +130,7 @@ px4_add_module(
 		sensors/flow.cpp
 		sensors/gnss.cpp
 		sensors/mag.cpp
+		sensors/beep.cpp
```

Comment:
Can you please move this above and make sure it is alphabetically ordered?

Reply:
ok